docs/guides/ORCHESTRATION_CHAT_COMPLETION.md

```diff
@@ -208,3 +208,19 @@ OrchestrationAiModel customGPT4O =
                 "presence_penalty", 0))
         .withVersion("2024-05-13");
 ```
+
+### Using a Configuration from AI Launchpad
+
+In case you have created a configuration in AI Launchpad, you can copy or download the configuration as JSON and use it directly in your code:
+
+```java
+var configJson = """
+    ... paste your configuration JSON in here ...
+    """;
+
+var prompt = new OrchestrationPrompt(Map.of("your-input-parameter", "your-param-value"));
+
+new OrchestrationClient().executeRequestFromJsonModuleConfig(prompt, configJson);
```

Comment:
Your suggested code change results in the following two methods


```java
OrchestrationClient client = new OrchestrationClient();

// execute completion request (exist)
CompletionPostRequest requestDto;
CompletionPostResponse responseDto = client
  .executeRequest(
    requestDto);

// execute prompt (exist)
OrchestrationPrompt promptConvenience;
OrchestrationModuleConfig configConvenience;
OrchestrationChatResponse responseConvenience = client
  .executeRequest(
    promptConvenience,
    configConvenience);

// execute prompt (new)
OrchestrationPrompt promptConvenience;
String configJson;
OrchestrationChatResponse responseConvenience = client
  .executeRequestFromJsonModuleConfig(
    promptConvenience,
    configJson);
```

(Question)

Is it not possible to derive reuse existing method and allow for `OrchestrationModuleConfig` to be created from `String configJson`? That would look a little cleaner, maybe(?)

Reply:
I also thought about this approach. But unfortunately not, because the JSON string is expected to contain things not yet supported by `OrchestrationModuleConfig` or it's individual config objects, i.e. throughout the entire JSON. That includes cases of `oneOf`, e.g. TemplateRef.